### ISSUE TITLE
[Challenges]Apply here!

### ISSUE THREAD

**Issue by jimmy3dita** [nearprotocol/stakewars#350]
Do you want to earn extra tokens?
We have contribution opportunities in the challenges section of Stake Wars. 

How does it work?
Normally, it's sufficient to create a tutorial in the form of a video, a blogpost or Github document, that shows how to complete the challenge. These guides will help other validators and will help us identify you as a great candidate to run TestNet and MainNet validator nodes.

Contributions in other languages are encouraged but considered on a case by case basis. Updates to these guides, which may be necessary to reflect new releases of the tools involved, will be paid additional tokens at each revision, up to once per month, until Stake Wars will be over.

Once your work is done, reviewed and accepted, it will be added to the list on these challenges. Please note that rewards in tokens will need to pass basic KYC checks from NEAR Foundation, and comply with your local regulations.

Reply below, specifying:
- which contribution you want to do, and the link to the challenge
- the type of document you will release
- the date when you will publish the content (you can postpone a few days, no worries)

Once your reply is posted, we will evaluate the proposal, agree privately on the scope and the acceptance criteria, and you'll be ready!

If you see the reaction 👀 I've seen this
If you see the reaction 👍 you got queued for the betanet tokens

**Comment by htafolla** [comment]
Challenge Accepted! How to compile nearcore: https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#list-of-available-contributions

PR on Stakewars II https://github.com/nearprotocol/stakewars#quickstart
PR https://docs.near.org/docs/validator/staking
PR https://docs.near.org/docs/local-setup/running-testnet

Due Date: Wed, June 3rd

**Comment by navneetlakra** [comment]
Challenge001 - Accepted. 
https://github.com/mutedtommy/near-docs/blob/master/compile-nearcore.md

I have created a document which lists steps to checkout the nearcore beta branch and build from there. The document also contains the steps to deploy a staking pool contract using [Staking Pool Factory UI](https://near-examples.github.io/staking-pool-factory/).

**Comment by narniec** [comment]
Challenge Accepted! 

- Russian translate "Enroll your node" 
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md
- Guide
- Due Date: Wed, June 3-4rd

**Comment by jimmy3dita** [comment]
Thanks guys! All contributions are accepted, please link below your finished document, so I can review the docs internally, and unlock the contribution

**Comment by navneetlakra** [comment]
Link to "How to Compile nearcore from beta" document: https://github.com/mutedtommy/near-docs/blob/master/compile-nearcore.md

**Comment by julypdx** [comment]
How to Compile nearcore from beta - Accepted.

https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#contribution-opportunities

- Chinese  "How to compile nearcore"

Due Date: Wed, June 3-4rd

**Comment by navneetlakra** [comment]
> Thanks guys! All contributions are accepted, please link below your finished document, so I can review the docs internally, and unlock the contribution

Link to "How to Compile nearcore from beta" document: https://github.com/mutedtommy/near-docs/blob/master/compile-nearcore.md

**Comment by navneetlakra** [comment]
Challenge Accepted - Explain Near Keys

https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#list-of-available-contributions

Due Date: 4-5 June

**Comment by julypdx** [comment]
> Thanks guys! All contributions are accepted, please link below your finished document, so I can review the docs internally, and unlock the contribution

Link to` Chinese "How to compile nearcore"` https://github.com/julypdx/near-docs/blob/master/COMPILE-NEARCORE_cn.md

**Comment by jimmy3dita** [comment]
Contributions under review:
@htafolla 
@navneetlakra 

Challenges:
All the current applications are accepted. I am reviewing the process to provide more scalability, will edit the upper thread with additional instructions later today.

**Comment by htafolla** [comment]
Ok. I was a little over-ambitious. I got this one done so far. 😄 

PR https://docs.near.org/docs/validator/staking

https://github.com/htafolla/docs/blob/master/docs/validator/staking.md

Let me know if it looks good enough to push the PR!

**Comment by narniec** [comment]
> Thanks guys! All contributions are accepted, please link below your finished document, so I can review the docs internally, and unlock the contribution
https://medium.com/@narniec2020/%D0%B7%D0%B0%D0%BF%D1%83%D1%81%D0%BA-%D0%BD%D0%BE%D0%B4%D1%8B-near-%D0%B2-%D1%82%D0%B5%D1%81%D1%82%D0%BE%D0%B2%D0%BE%D0%B9-%D1%81%D0%B5%D1%82%D0%B8-betanet-a4d58c9a7bb0
Russian guide how install nearcore + configure stacking pool and become validator. Unfortunately, I could not show everything on a live example are not enough NEAR :)
But my pool stacking account "TOP" validating  successfully work, through a deployed contract.
I chose the "Enroll your node", but i wanted to try to do more, I will continue
Despite some problems with the network, I'm trying to make a contribution somehow!

**Comment by masknetgoal634** [comment]
Challenge Accepted:
 - Monitor uptime
 - Manage the pool

Due Date: June 12

**Comment by NAKsir-melody** [comment]
Translate https://github.com/near/docs/blob/master/docs/validator/staking.md
into korean.

PR: https://github.com/near/docs/pull/385

**Comment by jimmy3dita** [comment]
Updates so far:
- Congrats @navneetlakra @georgec138 and @narniec, your contributions are included!
- Great work by @htafolla , added your contributions too (PR still in progress)
- Accepted the application by @masknetgoal634 and @NAKsir-melody 

Additional note: I staked 15k betanet tokens to @navneetlakra 's pool, send me a message on Discord to receive the same

**Comment by georgec138** [comment]
Challenge Accepted - Enroll your node
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md
- Github document & Video 

Due Date: June 12

**Comment by imnisen** [comment]
Hi, I want to apply these challenges:

- [Run a monitoring platform](https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge003.md#list-of-available-contributions)

- [Send alerts](https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge003.md#list-of-available-contributions)

- Type: Github document 
- Due Date: June 16

**Comment by NAKsir-melody** [comment]
We've done. setup grafana & send mail 
Validator: @staking.dsrv.betanet
Master account: @dsrv.betanet

![image](https://user-images.githubusercontent.com/21022937/84143385-72ba1900-aa91-11ea-8a81-508f8d81ed35.png)

**Comment by masknetgoal634** [comment]
>Monitor uptime - Github document
https://github.com/masknetgoal634/near-prometheus-exporter/blob/master/guide/GUIDE.md

**Comment by htafolla** [comment]
> > Monitor uptime - Github document
> 
> A little guide completed:
> https://github.com/masknetgoal634/near-prometheus-exporter/blob/master/guide/GUIDE.md

This is great work! How can we run this outside of Docker?

**Comment by lorriz** [comment]
1. I want to make a guide (medium) on how to transfer the node to another server, without stopping the validator with the nearcore build on it. Can it be attributed to "How to compile nearcore"?
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md
Dates: 18 June

2. Enroll your node (Medium)
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md
Dates: 19 June

**Comment by masknetgoal634** [comment]
>This is great work! How can we run this outside of Docker?
It's good idea, I will try it.

**Comment by georgec138** [comment]
> Challenge Accepted - Enroll your node
> https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md
> 
> * Github document ~~& Video~~
> 
> Due Date: June 12

#### Link to `Enroll your node` Github document
https://github.com/georgec138/near-docs/blob/master/Enroll-your-node_cn.md

**Comment by narniec** [comment]
"Explain NEAR keys" russian guide
https://medium.com/@narniec2020/near-shell-backup-%D0%BA%D0%BB%D1%8E%D1%87%D0%B5%D0%B9-b30e8ed1d39d
And youtube where i recover  my secure keys  https://www.youtube.com/watch?v=VHtzBjbmzT0&feature=youtu.be

**Comment by masknetgoal634** [comment]
>A little guide completed:
https://github.com/masknetgoal634/near-prometheus-exporter/blob/master/guide/GUIDE.md

Updated version ^

**Comment by manferber** [comment]
Challenge Accepted - Monitor your node
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge003.md

Medium guides on how to monitor the validator-node with no-docker Prometheus and Grafana in English and Russian. 

Due Date: June 15 (GMT)

**Comment by marco-sundsk** [comment]
Challenge Accepted! Run a monitoring platform: https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge003.md

I wrote a blog to describe how to analyse log and PRC from acratch, in English and Chinese.
For English version, see:
https://github.com/marco-sundsk/NEAR_DOC_zhcn/blob/master/StakeWar/near_monitor.md
For Chinese version, see:
https://github.com/marco-sundsk/NEAR_DOC_zhcn/blob/master/StakeWar/near_monitor_cn.md

Due Date: Have done.

**Comment by narniec** [comment]
> > A little guide completed:
> > https://github.com/masknetgoal634/near-prometheus-exporter/blob/master/guide/GUIDE.md
> 
> Updated version ^
Thank you for the good guide.
I created Russian guide and add few moments and screenshot all work! 
https://medium.com/@narniec2020/%D0%BC%D0%BE%D0%BD%D0%B8%D1%82%D0%BE%D1%80%D0%B8%D0%BD%D0%B3-near-%D1%87%D0%B5%D1%80%D0%B5%D0%B7-grafana-prometheus-18a8b0f74dfb

**Comment by bonsfi** [comment]
Please find detailed guide on how to run a node for Stakewars in Spanish:
https://medium.com/@bonsfi/como-crear-un-nodo-validador-en-near-protocol-6095f5cb491d

Super simple guide :)

**Comment by wjdfx** [comment]
Challenge Accepted!
1、https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#list-of-available-contributions
Explain NEAR keys （Chinese）

Due Date: June 20

2、https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#list-of-available-contributions
Enroll your node (Chinese）
Enroll your node (Chinese）
Monitor uptime (Chinese）
Use Ledger Nano S (Chinese）

Due Date: June 20

3、https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge003.md#list-of-available-contributions
Run a monitoring platform (Chinese）
Send alerts (Chinese）

Due Date: June 22

**Comment by georgec138** [comment]
Challenge Accepted - Manage the node
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md
Github doc chinese

Due Date: right now
Link to Manage the pool :https://github.com/georgec138/near-docs/blob/master/Manage-the-pool_cn.md

**Comment by jimmy3dita** [comment]
@bonsfi , @wjdfx and @georgec138 contributions under review.
All the others: I'm going to add your contributions to the respective challenge files, and will send the BetaNet tokens to your pools asap

**Comment by lorriz** [comment]
> 1. I want to make a guide (medium) on how to transfer the node to another server, without stopping the validator with the nearcore build on it. Can it be attributed to "How to compile nearcore"?
>    https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md
>    Dates: 18 June
> 2. Enroll your node (Medium)
>    https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md
>    Dates: 19 June

First one: https://medium.com/@nodesanna/stake-wars-challenge-001-d87d207db948

**Comment by Vasya-kripto** [comment]
Hello! I want to make these tutorials:
1. Explain NEAR keys (English) 23.06 (https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md)
2. Manage the pool (English) 25.06 (https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md)

**Comment by narniec** [comment]
Hello! 
I created 2 Russian guide for Stake Wars Challenge 002
["Enroll your node"](https://medium.com/@narniec2020/%D1%80%D0%B5%D0%B3%D0%B8%D1%81%D1%82%D1%80%D0%B0%D1%86%D0%B8%D1%8F-%D0%BD%D0%BE%D0%B4%D1%8B-%D0%B2-%D1%81%D0%B5%D1%82%D0%B8-near-84c7df7ee701) 
["Manage the pool"](https://medium.com/@narniec2020/%D1%83%D0%BF%D1%80%D0%B0%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5-%D1%81%D1%82%D0%B5%D0%B9%D0%BA%D0%B8%D0%BD%D0%B3-%D0%BF%D1%83%D0%BB%D0%BE%D0%BC-%D1%81%D0%B5%D1%82%D0%B8-near-3bbc92d54635)

Stefano, you accidentally added me with the wrong link. P
I don't know CN :P
![stef](https://user-images.githubusercontent.com/64380122/85211068-37033700-b34e-11ea-9ba5-1e0b650343d3.PNG)

**Comment by cryptoenthu1** [comment]
I have completed Challenge 001 and Challenge 002. Running Staking pool id 
beeram527-stakingpool in betanet (Account id : @beeram527.betanet).

May 25th 2020, Challenge 001 Deploy your node and your staking pool
May 25th 2020, Challenge 002 Become a validator and manage your seat

**Comment by georgec138** [comment]
Challenge Accepted - Explain NEAR keys
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md

Due Date: right now
Link to Explain NEAR keys：https://github.com/georgec138/near-docs/blob/master/Explain-NEAR-keys_cn.md

**Comment by imnisen** [comment]
> Hi, I want to apply these challenges:
> 
> * [Run a monitoring platform](https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge003.md#list-of-available-contributions), Github document
> * [Send alerts](https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge003.md#list-of-available-contributions) , Github document
> * Due Date: June 16

Sorry for the delay.

At origin, We planned to write a Prometheus exporter to monitor the near RPC calls easier. However, as we do half the job, we found that there has been a great one: [near-prometheus-exporte](https://github.com/masknetgoal634/near-prometheus-exporter)  . So we decided to use it directly.

My colleague at HashQuark wrote a docker-compose file to make it easier to deploy than the origin repo. Now users can deploy with one command `docker-compose up -d` after setting the config correctly. That's super simple!

Besides the origin great guide by @masknetgoal634, We also write two guides here and plan to update them as needed:

Run a monitoring platform: https://github.com/dolphintwo/near-prometheus-exporter/blob/master/guide/GUIDE-compose.md (Chinese version)

Send alerts: https://github.com/dolphintwo/near-prometheus-exporter/blob/master/guide/GUIDE-alert.md (Chinese version)

Any questions or requests are welcome!

From validator: c1.hashquark

**Comment by bonsfi** [comment]
Just created a post in spanish explaining how keys work and how to get a transaction done from another server without having to login.
Thanks Near Team!
From validator:nearcola

**Comment by dreamstaker** [comment]
> 1. I want to make a guide (medium) on how to transfer the node to another server, without stopping the validator with the nearcore build on it. Can it be attributed to "How to compile nearcore"?
>    https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md
>    Dates: 18 June
> 2. Enroll your node (Medium)
>    https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md
>    Dates: 19 June

I'v wrote the second one:
https://medium.com/@nodesanna/from-dreamstaker-betanet-valeraverim-staking-pool-f846676192a6

**Comment by narniec** [comment]
Hello :)
Stake Wars Challenge 004
["Monitor the Stake" Russian Guide
](https://medium.com/@narniec2020/%D0%BC%D0%BE%D0%BD%D0%B8%D1%82%D0%BE%D1%80%D0%B8%D0%BD%D0%B3-%D1%86%D0%B5%D0%BD%D1%8B-%D0%BC%D0%B5%D1%81%D1%82%D0%B0-%D0%B2-near-39ca567b6049)

**Comment by jimmy3dita** [comment]
@Vasya-kripto and @cryptoenthu1 go on with the contributions, there's still room =)
@dreamstaker , @narniec , @georgec138  , @imnisen and @bonsfi I added your contributions to pr #515

**Comment by jimmy3dita** [comment]
@georgec138 and @masknetgoal634 I can't figure out which pool you are using, let me know where I have to stake the 15,000 tokens =)

**Comment by narniec** [comment]
> @Vasya-kripto and @cryptoenthu1 go on with the contributions, there's still room =)
> @dreamstaker , @narniec , @georgec138 , @imnisen and @bonsfi I added your contributions to pr #515

Thank you!
I also created guide:  [Enroll your node](https://medium.com/@narniec2020/%D1%80%D0%B5%D0%B3%D0%B8%D1%81%D1%82%D1%80%D0%B0%D1%86%D0%B8%D1%8F-%D0%BD%D0%BE%D0%B4%D1%8B-%D0%B2-%D1%81%D0%B5%D1%82%D0%B8-near-84c7df7ee701), [Send alerts](https://medium.com/@narniec2020/%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D0%BD%D0%B8%D0%B5-%D0%B8-%D0%BE%D1%82%D0%BF%D1%80%D0%B0%D0%B2%D0%BA%D0%B0-%D1%83%D0%B2%D0%B5%D0%B4%D0%BE%D0%BC%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5-%D0%B2-grafana-b697f3e92cc4)  but i don't see myself in the lists, I did something wrong, or i do too much :D

**Comment by georgec138** [comment]
> @georgec138 and @masknetgoal634 I can't figure out which pool you are using, let me know where I have to stake the 15,000 tokens =)

sorry for my mistake
From validator: zpool

**Comment by masknetgoal634** [comment]
> @georgec138 and @masknetgoal634 I can't figure out which pool you are using, let me know where I have to stake the 15,000 tokens =)

My validator pool: jazza

**Comment by Viacheslav198** [comment]
Challenge Accepted!
Stake Wars Challenge 004
Italian guide "Near node monitoring"
https://github.com/Viacheslav198/grafana-prometheus/blob/master/alert.md

**Comment by jimmy3dita** [comment]
@Viacheslav198 Contribution accepted, keep up with the good work =)
@georgec138 I've sent the 15k tokens for your contribution. I've mistakenly sent them twice, so I had to unstake some of them.
@masknetgoal634 same as above, but done right :D

**Comment by masknetgoal634** [comment]
Challenge004: 
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge004.md#list-of-available-contributions

1. Monitor the Stake:
 https://github.com/masknetgoal634/near-go-warchest/blob/master/README.md

2. Release the Warchest Bot
https://github.com/masknetgoal634/near-go-warchest

^ Actually not tested well the warchest bot because lack of enough stake for 2 seats.

**Comment by Neozaru** [comment]
Hello.
For the Challenge 004, contribution 2 - warchest bot :
https://github.com/Neozaru/near-warchest-bot

**Comment by Vasya-kripto** [comment]
> Hello! I want to make these tutorials:
> 
> 1. Explain NEAR keys (English) 23.06 (medium) (https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md)
> 2. Manage the pool (English) 25.06 (medium) (https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md)

Sorry for delay! 
I'v made one: https://medium.com/@orang3club/explain-near-keys-737162914413
The second will be in coming days!

**Comment by imnisen** [comment]
> @Vasya-kripto and @cryptoenthu1 go on with the contributions, there's still room =)
> @dreamstaker , @narniec , @georgec138 , @imnisen and @bonsfi I added your contributions to pr #515

Hi, @jimmy3dita ,  as I cannot find our validator c1.hashquark at [List of Validators who already completed this challenge](https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge003.md#list-of-validators-who-already-completed-this-challenge-updated-jun-9), will you stake extra token to our stake pool as we have already finished the monitor and set the aleart. If not, what else should I do? Thanks!

**Comment by 48cfu** [comment]
Release the Warchest Bot in python
[Challenge 004](https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge004.md).
Delivery date: 3rd July 2020
Beta version repo: [warchest-bot-python](https://github.com/48cfu/near-warchest-bot)

**Comment by gaia** [comment]
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge004.md

Warchest Bot in less than 100 lines of bash - nothing to install. Run this script via cron, every 10 minutes should be enough.

https://gist.github.com/gaia/ae0d6eee566f6a7067b1d62b631a2f20

Validator: freshnears

**Comment by jimmy3dita** [comment]
@48cfu and @gaia awesome work. Will include you in this week updates and assign you the tokens.
@Vasya-kripto good guide, I'll include it in the list. Don't forget to include the name of your pool here somewhere, as I delegate 15k extra tokens to contributors.

@imnisen I tried to delegate tokens, but my script found that your pool is not locked and refused to do so :)

**Comment by Vasya-kripto** [comment]
> @48cfu and @gaia awesome work. Will include you in this week updates and assign you the tokens.
> @Vasya-kripto good guide, I'll include it in the list. Don't forget to include the name of your pool here somewhere, as I delegate 15k extra tokens to contributors.
> 
> @imnisen I tried to delegate tokens, but my script found that your pool is not locked and refused to do so :)

If it concerned me, then my staking pool is **somebody**

**Comment by imnisen** [comment]
> @48cfu and @gaia awesome work. Will include you in this week updates and assign you the tokens.
> @Vasya-kripto good guide, I'll include it in the list. Don't forget to include the name of your pool here somewhere, as I delegate 15k extra tokens to contributors.
> 
> @imnisen I tried to delegate tokens, but my script found that your pool is not locked and refused to do so :)

Ah, missing it!  Now I have deleted the access key.

**Comment by gaia** [comment]
> my script found that your pool is not locked and refused to do so

@jimmy3dita how do we check whether a contract has been locked? shouldn't the cli at least warn when it hasn't been locked?

**Comment by marshall-d-teach** [comment]
Hi, I want to apply this.

Challenge1:  [How to Compile nearcore from beta](https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md) 

Due Date: Finished

Guide: https://github.com/marshall-d-teach/near-doc/blob/master/create-validator.md

Validator:  stakepool2.xiaoshen.betanet

**Comment by eorituz** [comment]
Application for Challange 004 easy to adapt warchestbot in python3.6+
Github link: https://github.com/eorituz/near_warchest
status: Finished

**Comment by Vasya-kripto** [comment]
> Hello! I want to make these tutorials:
> 
> 1. Explain NEAR keys (English) 23.06 (medium) (https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md)
> 2. Manage the pool (English) 25.06 (medium) (https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md)

And another one:
https://medium.com/@orang3club/manage-your-staking-pool-80709494cc63
I won't be late next time :-)

**Comment by jimmy3dita** [comment]
@narniec updated the docs with the correct contribution links

@Viacheslav198 I can't associate your guides with the available contributions. What I could find is a translation of the challenge text or contributions from other users. Furthermore, I am Italian and I can recognize that some of these guides have been made using automated translation ("server di picchettamento" is a good example), so I except you to improve the quality of them, or withdraw your contributions. The only one that is acceptable is the challenge 003, about configuring Grafana.

@marshall-d-teach, @eorituz, and @Vasya-kripto added your contributions too. Keep up with the good work!

**Comment by jimmy3dita** [comment]
> > my script found that your pool is not locked and refused to do so
> 
> @jimmy3dita how do we check whether a contract has been locked? shouldn't the cli at least warn when it hasn't been locked?

You can open the account id using the explorer, or issue `near keys <ACCOUNT_ID>` from near-shell.
The pool `freshnears` results as locked: https://explorer.betanet.near.org/accounts/freshnears

**Comment by narniec** [comment]
> @narniec updated the docs with the correct contribution links
> 
> @Viacheslav198 I can't associate your guides with the available contributions. What I could find is a translation of the challenge text or contributions from other users. Furthermore, I am Italian and I can recognize that some of these guides have been made using automated translation ("server di picchettamento" is a good example), so I except you to improve the quality of them, or withdraw your contributions. The only one that is acceptable is the challenge 003, about configuring Grafana.
> 
> @marshall-d-teach, @eorituz, and @Vasya-kripto added your contributions too. Keep up with the good work!
> 
> This is the pending PR: #532
Ah, I also created a guide(Enroll your node, Send alerts, Use Ledger Nano S), but they weren't added in tables challenges, i meant this moment :)
everything else was correct :)

**Comment by Viacheslav198** [comment]
Challenge Accepted!

Challenge 002
Russian guide"Use Ledger Nano S"
https://github.com/Viacheslav198/near/blob/master/ledger.md

Date: 02.07

**Comment by Lalit15** [comment]
Challenge Accepeted

Challenge 003
English guide for setting up Grafana and Prometheus. Current guide shared by masknetgoal634 lacks a few critical steps to complete grafana setup and those have been added in this link
https://medium.com/@coolphil15/setting-up-monitoring-on-near-node-via-grafana-prometheus-fd641ff82aeb

Date:02.07

**Comment by imnisen** [comment]
Validator: c1.hashquark

Challenge004:
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge004.md#list-of-available-contributions

Monitor the Stake:
https://github.com/imnisen/near-warchest/blob/master/README.md

Release the Warchest Bot:
https://github.com/imnisen/near-warchest

**Comment by marshall-d-teach** [comment]
> Hi, I want to apply this.
> 
> Challenge1: [How to Compile nearcore from beta](https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md)
> 
> Due Date: Finished
> 
> Guide: https://github.com/marshall-d-teach/near-doc/blob/master/create-validator.md
> 
> Validator: stakepool2.xiaoshen.betanet

Another in challenge001.

https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#list-of-available-contributions

Explain NEAR keys:
https://github.com/marshall-d-teach/near-doc/blob/master/explain-near-keys.md

Date: Finished

**Comment by marshall-d-teach** [comment]
Validator: stakepool2.xiaoshen.betanet

Challenge002, Enroll your node
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md

github doc: https://github.com/marshall-d-teach/near-doc/blob/master/enroll-your-node.md

Due Date: Finished

**Comment by jimmy3dita** [comment]
@Viacheslav198 @Lalit15 @marshall-d-teach I added your contributions to the challenge files.

@imnisen your contribution to the monitoring of the stake is limited, so by now I added only the warchest bot link.

**Comment by wjdfx** [comment]
Validator: @bitcat.stakehouse.betanet
Master account: @catcatcat.test

Challenge 004
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge004.md#list-of-available-contributions
Monitor the Stake(Chinese)
https://www.jianshu.com/p/18b5d159d73a 如何查看和调整NEAR的验证人质押状态

**Comment by imnisen** [comment]
Validator: c1.hashquark

Challenge001:
https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#list-of-available-contributions

contribution2 - Explain NEAR keys[chinese version]:
https://github.com/imnisen/near-stakewars-guide/blob/master/challenge001-explain-near-keys.org

Date: Done Now.

**Comment by Viacheslav198** [comment]
Challenge Accepted!

Challenge 001
Russian guide "Key generation and backup"
https://github.com/Viacheslav198/near/blob/master/keys.md

Date: 07.07

**Comment by masknetgoal634** [comment]
- Monitor uptime
  https://github.com/masknetgoal634/near-prometheus-exporter/blob/master/guide/GUIDE.md
  Status: Updated 
  Note: Thanks @Lalit15 updated following your guide

- Release the Warchest Bot
  https://github.com/masknetgoal634/near-go-warchest
  Status: Finished

**Comment by WilliamGuozi** [comment]
Challenge004:
https://github.com/WilliamGuozi/near-monitor/blob/master/near-monitor.sh

**Comment by qwerspe** [comment]
Chanllenge 001 "How to compile nearcore" https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#list-of-available-contributions

Medium: https://medium.com/@qwerspe/compile-nearcore-and-create-validator-338fbf471ec0

Status: Finished

Pool id: p.qwerspe.betanet

**Comment by Lalit15** [comment]
Challenge001:  How to compile nearcore
Complete guide that include Near wallet, Near CLI and Near explorer deploy and understand how node works. 
With Near CLI snapshots at every step for easy reference.

https://medium.com/@coolphil15/launch-near-node-on-near-betanet-test-network-4fcd810a71b5

Date: 10 July 2020

**Comment by Lalit15** [comment]
Challenge001:  Explain NEAR keys
Simple step by step guide to authorize actions on near node without logging to your account in english.

https://medium.com/@coolphil15/how-to-generate-near-keys-using-near-shell-and-authorize-access-6b114003e9d

Date: 10 July 2020

Validator: goodstake3.lalitnear.betanet
--

**Comment by jimmy3dita** [comment]
PSA: we are going to close this thread, and migrate all applications to https://portal.near.org.
If you haven't already, fill up the Challenge 004 form [here](https://nearprotocol1001.typeform.com/to/x4Bval), so we can send you an invitation to the platform.

**Comment by qwerspe** [comment]
Challenge001:  How to compile nearcore

https://medium.com/@qwerspe/compile-nearcore-and-create-validator-338fbf471ec0

Date: 11 July

Validator: @p.qwerspe.betanet

@jimmy3dita  In order to fill the form, do I need to finish the challenge 4 or deploy a testnet node first? It needs the testnet pool id.

**Comment by bonsfi** [comment]
My spanish contribution guide on how to participate in stakewars has not been included in challenge 001 contributions. Could you please include it and assign bonust tokens? Thanks :)
https://medium.com/@bonsfi/como-crear-un-nodo-validador-en-near-protocol-6095f5cb491d

validator:nearcola

**Comment by bonsfi** [comment]
Just produced Spanish guide on how to connect Ledger with Near Protocol for Challenge 002
validator:nearcola
near portal user:bonsfi

**Comment by jimmy3dita** [comment]
> My spanish contribution guide on how to participate in stakewars has not been included in challenge 001 contributions. Could you please include it and assign bonust tokens? Thanks :)
> https://medium.com/@bonsfi/como-crear-un-nodo-validador-en-near-protocol-6095f5cb491d
> validator:nearcola
> near portal user:bonsfi

It's a great job! But I can't see how it can be included in one of the available contributions below

| ID | Abstract | Description |
| --- | --- | --- |
| 1 | How to compile nearcore | Create a tutorial, in the form of a video, a blogpost or Github document, that explains how to clone, compile and run `nearcore` from the beta branch, and prove that you can become validator with the process. Contributions in other languages are encouraged, but considered case by case basis. Updates to this guide, reflecting any updates of the tools involved, will be paid a % of the initial bounty per each revision, up to once per month, until Stake Wars is over. |
| 2 | Explain NEAR keys | Explain in the form of a video, a blogpost or Github document how `near login` works, how the authorization keys are generated, and how you can manage their backup and restore properly. Show in the guide the proof that you were able to run `near shell` on a different machine, without authorizing a new key. Contributions in other languages are encouraged, but considered on a case by case basis. Updates to this guide, reflecting any updates of the tools involved, will be paid a % of the initial bounty per each revision, up to once per month, until Stake Wars is over. |

**Comment by Vasya-kripto** [comment]
> @48cfu and @gaia awesome work. Will include you in this week updates and assign you the tokens.
> @Vasya-kripto good guide, I'll include it in the list. Don't forget to include the name of your pool here somewhere, as I delegate 15k extra tokens to contributors.
> 
> @imnisen I tried to delegate tokens, but my script found that your pool is not locked and refused to do so :)


Will my guide be included in the list here: https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md or was it credited with only 15,000 test tokens?

**Comment by bonsfi** [comment]
Just did this SPANISH guide on how to monitor stake for stakewars challenge 4:
https://medium.com/@bonsfi/como-monitorear-el-stake-de-tu-validador-en-near-protocol-d709326cf6ff?sk=09f62413b029762a549b5596c9543a4c

Hope it's useful. I'm bonsfi and my validator is Nearcola

**Comment by jimmy3dita** [comment]
> 
> Will my guide ([#350 (comment)](https://github.com/nearprotocol/stakewars/issues/350#issuecomment-650153786)) be included in the list here: https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md or did it just credited with 15,000 test tokens?

You will receive 15k BetaNet tokens, I simply need to know what is the name of your pool. Do you confirm it is`somestaking.stakingpool`?

@bonsfi will be added in the next PR

**Comment by Vasya-kripto** [comment]
> > Will my guide ([#350 (comment)](https://github.com/nearprotocol/stakewars/issues/350#issuecomment-650153786)) be included in the list here: https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md or did it just credited with 15,000 test tokens?
> 
> You will receive 15k BetaNet tokens, I simply need to know what is the name of your pool. Do you confirm it is`somestaking.stakingpool`?
> 
> @bonsfi will be added in the next PR

I already received 15,000 tokens earlier in my betanet pool - "somebody". And the pool - somestaking.stakingpool is my account in the testnet network. Now I just wanted to clarify whether my guide will be included in the "List of available contributions" in the "Explain NEAR keys" item or not. Earlier You wrote that you would include it in the list, but it is not there)

**Comment by minstr22** [comment]
We've done. setup grafana & send mail
Validator: @paulainsley1betanet.stakehouse.betanet
Master account: @aulainsley1.betanet

**Comment by minstr22** [comment]
email alert send.

**Comment by 48cfu** [comment]
> @48cfu and @gaia awesome work. Will include you in this week updates and assign you the tokens.
> @Vasya-kripto good guide, I'll include it in the list. Don't forget to include the name of your pool here somewhere, as I delegate 15k extra tokens to contributors.
> 
> @imnisen I tried to delegate tokens, but my script found that your pool is not locked and refused to do so :)

Hi Stefano! I'm not sure why my contribution wasn't included in the list of contributions...

**Comment by jimmy3dita** [comment]
@Vasya-kripto and @48cfu corrected in this week's updates.

PSA: this thread is now closed. Please use the NEAR Portal here: https://portal.near.org/topic/validator
If you don't have an account yet, use [this form](https://nearprotocol1001.typeform.com/to/x4Bval) to submit your validator information even if you haven't completed all the challenges, so we can send you the invitation to the portal.

In case of issues with this transition, feel free to reach out on Discord, via https://near.chat.

**Comment by imnisen** [comment]
> Validator: @c1.hashquark
> 
> Challenge001:
> https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge001.md#list-of-available-contributions
> 
> * contribution2 - Explain NEAR keys[chinese version]:
>   https://github.com/imnisen/near-stakewars-guide/blob/master/challenge001-explain-near-keys.org
> 
> Challenge002:
> https://github.com/nearprotocol/stakewars/blob/master/challenges/challenge002.md#list-of-available-contributions
> 
> * contribution1 - Enroll your node[chinese version]:
>   https://github.com/imnisen/near-stakewars-guide/blob/master/challenge002-enroll-your-node.org
> * contribution2 - Manage the pool[chinese version]:
>   https://github.com/imnisen/near-stakewars-guide/blob/master/challenge002-manage-the-pool.org
> * contribution3 - Monitor uptime[chinese version]:
>   https://github.com/imnisen/near-stakewars-guide/blob/master/challenge002-monitor-uptime.org
> * contribution4 - Use Ledger Nano S[chinese version]:
>   https://github.com/imnisen/near-stakewars-guide/blob/master/challenge002-use-ledger-nano-s.org
> 
> Date: Done Now.

Hi Jimmy, would this be accepted?

**Comment by Cryptomilion** [comment]
Validator: @cryp
Master account: @cryptomilion.betanet
Challenge001: Accepted! Ukrainian guide: сompile nearcore and deploy stacking pool

https://www.youtube.com/watch?v=9YlTCYe13eY

https://medium.com/@shiverov/%D0%B7%D0%B0%D0%BF%D1%83%D1%81%D0%BA-%D0%BD%D0%BE%D0%B4%D0%B8-nearcore-near-%D0%B2-%D1%82%D0%B5%D1%81%D1%82%D0%BE%D0%B2%D1%96%D0%B9-%D0%BC%D0%B5%D1%80%D0%B5%D0%B6%D1%96-betanet-216415954c4e

**Comment by jimmy3dita** [comment]
please use [NEAR portal](https://portal.near.org) to report your contributions, and once approved do a pull request to the challenge file.

**Comment by dreamstaker** [comment]
> please use [NEAR portal](https://portal.near.org) to report your contributions, and once approved do a pull request to the challenge file.
> This thread is not followed anymore, so any contribution posted here will not be counted as valid.
> If you have problems joining the portal, contact me directly on Discord or Telegram, and I'll send you the invite again.

I guess I don't need to make a pull request, since they are already in the table. I thought I should just repeat the links to them here on the portal :)

How to compile nearcore by @dreamstaker (challenge001)
& 
Enroll your node by @dreamstaker (challenge002)

**Comment by bonsfi** [comment]
Hello, as Near required, I prepared two guides on how to use Ledger with Near. The guides go from setting your account from scratch to signing transactions with Ledger. Please finde them below:)
https://medium.com/@bonsfi/como-usar-leger-con-near-cli-8b9ed66f3b1d
https://medium.com/@bonsfi/how-to-use-ledger-with-near-cli-648d5d990517

**Comment by rozum-dev** [comment]
Validator: @coolpool.dimaro1.betanet
Master account: @dimaro1.betanet

Challenge001: Accepted! Explaining NEAR keys in Ukrainian

**Comment by rozum-dev** [comment]
Validator: @coolpool.dimaro1.betanet
Master account: @dimaro1.betanet

Challenge002: Accepted! Enrol your node in Ukrainian

https://medium.com/@dmytro.rozum/%D0%B7%D0%B0%D1%80%D0%B5%D1%94%D1%81%D1%82%D1%80%D1%83%D0%B9%D1%82%D0%B5-%D1%81%D0%B2%D0%BE%D1%8E-%D0%BD%D0%BE%D0%B4%D1%83-39be8334a512

**Comment by rozum-dev** [comment]
Validator: @coolpool.dimaro1.betanet
Master account: @dimaro1.betanet

Challenge003: Accepted! Setting up Monitoring on Near node via Grafana + Prometheus in Ukrainian

https://medium.com/@dmytro.rozum/%D0%BC%D0%BE%D0%BD%D1%96%D1%82%D0%BE%D1%80%D0%B8%D0%BC%D0%BE-%D1%81%D1%82%D0%B0%D0%BD-%D0%BD%D0%BE%D0%B4%D0%B8-near-node-%D0%B7-prometheus-%D1%82%D0%B0-grafana-1943da054df4

**Comment by savelev1** [comment]
Python script to manage your validator seat (Challenge 004)
RU/EN version here https://github.com/savelev1/near-warchest-bot

**Comment by savelev1** [comment]
Python script for automatically update nearcore (Challenge 005)
RU/EN version here https://github.com/savelev1/nearcore-updater

**Comment by anechoic-git** [comment]
validator: @anechoic.stakethouse.betanet
master account: @anechoic.betanet

**which contribution you want to do, and the link to the challenge**
Challenge 001: How to compile nearcore

**the type of document you will release**
Video: https://youtu.be/FCf-jTFtMcI

**the date when you will publish the content (you can postpone a few days, no worries)**
Today 9/12/2020

**Comment by 48cfu** [comment]
> validator: @anechoic.stakethouse.betanet
> master account: @anechoic.betanet
> 
> **which contribution you want to do, and the link to the challenge**
> Challenge 001: How to compile nearcore
> 
> **the type of document you will release**
> Video: https://youtu.be/FCf-jTFtMcI
> 
> **the date when you will publish the content (you can postpone a few days, no worries)**
> Today 9/12/2020

You should use the near portal to apply to challenges.
